Tests/UKHO.MaritimeSafetyInformation.Web.UnitTests/Controllers/RadioNavigationalWarningsAdminControllerTest.cs

```diff
@@ -95,6 +99,16 @@ public async Task WhenICallGetRadioNavigationWarningsForAdmin_ThenReturnViewAndV
             Assert.IsNotNull(((ViewResult)result).ViewData["Years"]);
         }
 
+        [Test]
+        public void WhenICallGetRadioNavigationWarningsForAdmin_ThenCheckIfUserHasCorrectRole()
+        {
+            object[] actualAttribute = _controller.GetType().GetCustomAttributes(typeof(AuthorizeAttribute), true);
+            object role = _controller.GetType().GetCustomAttributes(typeof(AuthorizeAttribute), true).GetValue(0);
```

Comment:
This can be refactored as below:
```suggestion
            object role = actualAttribute.GetValue(0);
```

Reply:
fixed